Alignment/CommonAlignmentProducer/bin/aligncond_split_iov.cc

```diff
@@ -1,185 +0,0 @@
-#include "CondCore/DBCommon/interface/DbSession.h"
```

Comment:
@ggovi @diguida  are you going to provide a tool allowing to split the IOVs suitable for v2? I believe this would be highly welcome from the tracker alignment group @mschrode @tlampen